code/game/objects/machinery/doors/multi_tile.dm

```diff
@@ -4,7 +4,7 @@
 
 /obj/machinery/door/airlock/multi_tile/close() //Nasty as hell O(n^2) code but unfortunately necessary
 	for(var/turf/T in locs)
-		for(var/obj/vehicle/multitile/M in T)
+		for(var/obj/vehicle/M in T)
 			if(M) return FALSE
```

Comment:
Does this need to exist? The airlock should assumingly be not closing if any dense object is blocking it

If it does need to exist then this internal loop should be replaced with
```
if(locate(/obj/vehicle) in T)
  return FALSE
```